schema_based_on_diagram.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE medical_histories (
+    id SERIAL PRIMARY KEY,
+    admitted_at TIMESTAMP,
+    patient_id INTEGER,
+    status VARCHAR
+);
```

Comment:
- [ ] Good job creating a database based on a diagram, but you should implement the  Join tables from many-to-many relationships. Kindly implement it 👍 

```sql
CREATE TABLE medical_histories_has_treatments (
//,,.
)
```

Reply:
Thanks for the information, I will make the requested changes.